devops/resources/run-app-permissions.sh

```diff
@@ -18,8 +18,37 @@ assume-role ${ROLE_ARN}
 cd $WORKSPACE/configuration/playbooks
 
 INVENTORY=$(./active_instances_in_asg.py --asg ${ENVIRONMENT}-${DEPLOYMENT}-worker)
+
 if [[ -n ${INVENTORY} ]]; then
-    ansible-playbook -i ${INVENTORY} manage_edxapp_users_and_groups.yml -e@$WORKSPACE/app-permissions/${ENVIRONMENT}-${DEPLOYMENT}-edxapp.yml --user ${USER} --tags manage-${JOB_TYPE}
+
+    if [[ -n "$USE_EXPERIMENTAL_SERVICE_VARIANT_GROUPS" && "$ENVIRONMENT" = stage ]]; then
+        ansible-playbook \
+            -i ${INVENTORY} \
+            manage_lms_groups.yml \
+            -e@$WORKSPACE/app-permissions/groups/lms.yml \
+            --user ${USER} \
+            --tags manage-${JOB_TYPE}
+        ansible-playbook \
+            -i ${INVENTORY} \
+            manage_cms_groups.yml \
+            -e@$WORKSPACE/app-permissions/groups/cms.yml \
+            --user ${USER} \
+            --tags manage-${JOB_TYPE}
+        ansible-playbook \
+            -i ${INVENTORY} \
+            manage_edxapp_users.yml \
+            -e@$WORKSPACE/app-permissions/${ENVIRONMENT}-${DEPLOYMENT}-edxapp.yml \
```

Comment:
I like this new naming scheme and think it will make it easier if we want to add this for ecommerce later on.